### DISCUSSION TITLE
Footer rigtht icon (backspace) and emoji CLDR short name

### DISCUSSION THREAD

**Discussion by KKTadena** [Aghajari/AXEmojiView#17] [Q&A]
Is it possible to edit / remove this icon ? 

Also is there a way we can get the CLDR short name of the emoji selected ? 

![image](https://user-images.githubusercontent.com/23197590/125466064-5ff617ec-186e-47d5-aaa2-bf71399993fd.png)

**Comment by Aghajari** [comment]
Hi, 

Sure, You can have access to the view by `emojiPager.getFooterRightView()`

And about "CLDR short name of the emoji", You should create a local db and search for emoji name by its unicode. or just search on AXEmojiView db for search, the first result could be what you are looking for.
